external-crates/move/crates/move-package/tests/test_runner.rs

```diff
@@ -114,6 +126,13 @@ impl Test<'_> {
     }
 }
 
+/// Return the dependencies contained in the file at `path`, if any
+fn load_implicits(path: &Path) -> Dependencies {
+    let deps_toml = fs::read_to_string(path).unwrap_or("# no implicit deps".to_string());
+    toml::from_str(&deps_toml)
```

Comment:
FWIW, I was suggesting something slightly different here, which was to re-use exactly the manifest parser that we use for `Move.toml` (which is a custom parser). It means a little bit more boilerplate when defining implicit deps, but you don't have to add the `Serialize`/`Deserialize` derives on all the related types (which we would like to avoid to prevent confusion about what the right way to serde these is).